auctions/english-auction/packages/scripts/README.md

```diff
@@ -0,0 +1,50 @@
+## Egnlish Auction Scripts
+
+This package has scripts to help integrate Sway contracts
+with the English Auction application. This makes it easier for developers to
+change contracts while building a nice UI
+
+### Execute
+
+```sh
+pnpm exec english-auction-scripts [command]
+```
+
+### Options
```

Comment:
1.) what do you mean providing the output into a file?
2.) I feel like the usage is explicit

Also I don't want to spend too much time messing around with this package as I stated earlier it will be phased into the ts sdk.